index.js

```diff
@@ -9,104 +9,105 @@ let x = "590516"; // create number manual
 const { urlAPI } = process.env;
 const urlAPILotto = `${urlAPI}`;
 
-// program
-//   .name("check-lotto-CLI")
-//   .description("You can check thai lottery and become a millionaire!")
-//   .version("1.0.0")
-//   .requiredOption("-d, --date <date>", "Date to check lottery")
-//   .requiredOption("-l, --lotto <lottery>", "Number lottery to check!")
-//   .action()
-
 const getdata = async (date) => {
-  console.log(date);
-  await axios.get(`${urlAPILotto}${date}`).then((response) => {
-    const numLottoData = Object.entries(response.data.data.prizes);
-    const arrayLottoFinal = [];
-    numLottoData.forEach((item) => {
-      let numLottoTemp = Object.entries(item);
-      let arrayTemp = [];
-      arrayTemp.push([numLottoTemp[0][1], Object.entries(numLottoTemp[1][1])]);
-      arrayLottoFinal.push([
-        arrayTemp[0][0],
-        arrayTemp[0][1][0][1],
-        arrayTemp[0][1][1][1],
-      ]);
-    });
+  const response = await axios.get(`${urlAPILotto}${date}`)
+  const numLottoData = Object.entries(response.data.data.prizes);
+  const arrayLottoFinal = [];
 
-    // change index to run 1-9
-    arrayLottoFinal[7][0] = "8";
-    arrayLottoFinal[8][0] = "9";
+  for (const item of numLottoData) {
+    let numLottoTemp = Object.entries(item);
+    let arrayTemp = [];
+    arrayTemp.push([numLottoTemp[0][1], Object.entries(numLottoTemp[1][1])]);
+    arrayLottoFinal.push([
+      arrayTemp[0][0],
+      arrayTemp[0][1][0][1],
+      arrayTemp[0][1][1][1],
+    ]);
+  }
```

Comment:
ช่วงข้อมูลตรงนี้ไม่ควรใช้ `.forEach()` ครับ มันอาจจะทำงานผิดช่วงและทำงานไม่ทันได้เนื่องจากมันเป็น Callback ดังนั้นควรใช้ `for of` ดังตัวอย่างนี้เแทน